src/index.js

```diff
@@ -0,0 +1,34 @@
+const BigNumber = require("bignumber.js");
+
+const { getInputOutputFiles, parseCsvFile, writeJson } = require("./utils");
+const GNO_ADDRESS = "0x6810e776880C02933D47DB1b9fc05908e5386b96";
+
+function toPayment(leaderBoardItem) {
+  const { trader, gno_estimation: gnoEstimation } = leaderBoardItem;
+
+  return {
+    amount: gnoEstimation,
+    tokenAddress: GNO_ADDRESS,
+    receiver: trader,
+  };
+}
+
+async function main() {
+  // Parse CSV
+  const [inputFile, outputFile] = getInputOutputFiles();
+  const results = await parseCsvFile(inputFile);
+
+  // Transform into payment object
+  const payments = results
+    .map(toPayment)
+    // Remove payment with amount zero
+    .filter((payment) => {
```

Comment:
Should we instead filter by a minimum threshold?

In order words, should we bother sending tiny amounts out? For example, smaller than 0.00001 GNO

Reply:
The script won't give you anything smaller of 0.0001

Happy to make the additional check if we think it's an issue. 
The query already ensures this

Reply:
Similar comment here: https://github.com/anxolin/challenge-payment/pull/1#discussion_r441024290